docs/samples/explanation/aix/Fetch_20newsgroups/README.md

```diff
@@ -0,0 +1,40 @@
+# Using AIX to get explanations for NewsGroups classifications
+
+This is another explanation example for text compare to [MNIST classifications (for image)](https://github.com/kserve/kserve/tree/master/docs/samples/explanation/aix/mnist).
+
+To deploy the inferenceservice with v1beta1 API
+
+`kubectl apply -f aix-explainer.yaml`
+
+Then find the url.
+
+`kubectl get inferenceservice`
```

Comment:
```suggestion
`kubectl get inferenceservice aix-explainer`
```

Reply:
@yuzisun I've signed-off and commit suggestion, thanks.